misc-utils/exch.c

```diff
@@ -26,14 +26,23 @@
 # define RENAME_EXCHANGE (1 << 1)
 #endif
 
-#if !defined(HAVE_RENAMEAT2) && defined(SYS_renameat2)
-static inline int renameat2(int olddirfd, const char *oldpath,
-			    int newdirfd, const char *newpath, unsigned int flags)
-{
-	return syscall (SYS_renameat2, olddirfd, oldpath, newdirfd, newpath, flags);
-}
+static inline int rename_exchange(int dirfd,
```

Comment:
I guess there can be a build environment where SYS_renameat2 is defined but RENAME_EXCHANGE is not.

How about adding the definition for RENAME_EXCHANGE?

```
#ifndef RENAME_EXCHANGE
#define RENAME_EXCHANGE		(1 << 1)	/* Taken from linux/include/uapi/linux/fs.h */
#endif
```

We can do the same for SYS_renameat2. However, I think it is over-killing.

Let's see how the maintainer says :-)